content/PGP/Card_edit.adoc

```diff
@@ -12,11 +12,18 @@ To configure the OpenPGP application on a YubiKey you can use GnuPG.
 
 == Prerequisites
 
-Make sure you have a recent YubiKey with firmware version 3.1.8 or
-later; use "lsusb -v" to find out.
+Installed the link:https://developers.yubico.com/yubikey-personalization/Manuals/ykpersonalize.1.html[Yubikey personalization tool].
 
-Make sure the device is in OTP/CCID or CCID mode, use the YubiKey
-personalization tool "ykpersonalize -m82" to switch.
+Verify that your YubiKey has a firmware version 3.1.8 or later.
+
+  $ ykpersonalize
+  Firmware version 4.3.1 Touch level 517 Program sequence 1
+  ...
+
+Make sure that your device is in OTP/CCID or CCID mode, you can use ykpersonalize to switch:
```

Comment:
This is actually outdated as any mode with CCID enabled should work (so it should include U2F modes as well). We've shipped YubiKeys with all transports enabled for some time, so for most people it should "just work", but it can still be an issue with older devices or devices where the user has manually switched the mode. If we give an example, it should probably be mode 6 (OTP+CCID+U2F) instead.